estate/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    'name': "Real Estate",
+    'license': 'LGPL-3',
+    'version': '1.0',
+    'depends': ['base'],
+    'author': "ngtpn",
+    'category': 'Category',
+    'description': """
+    Real Estate Advertisement module
+    """,
+    'data': [
```

Comment:
```suggestion
    'application': True,
    'data': [
```
You probably want to have application=True for this module. See the [manifest reference](https://www.odoo.com/documentation/19.0/developer/reference/backend/module.html)